src/pudl/transform/eia861.py

```diff
@@ -1091,8 +1239,8 @@ def transform(raw_dfs, eia861_tables=pc.pudl_tables["eia861"]):
     """
     # these are the tables that we have transform functions for...
     tfr_funcs = {
-        "service_territory_eia861": service_territory,
         "balancing_authority_eia861": balancing_authority,
+        "service_territory_eia861": service_territory,
         "sales_eia861": sales,
         "advanced_metering_infrastructure_eia861": advanced_metering_infrastructure,
         "demand_response_eia861": demand_response,
```

Comment:
Dynamic pricing should probably be in this?

Reply:
Oh yeah definitely. Did I remove it somehow?

Reply:
Oh it is there -- it's just outside of the excerpt being shown here. You can expand the excerpts with the little arrows at the top and bottom of them.

Reply:
Oh I see! Looks good then